nncf/quantization/passes.py

```diff
@@ -85,20 +65,65 @@ def remove_shapeof_subgraphs(
             shape_of_queue.extend(nncf_graph.get_next_nodes(node) + nncf_graph.get_previous_nodes(node))
 
     nncf_graph.remove_nodes_from(nodes_to_drop)
-    return nncf_graph
 
 
-def filter_constant_nodes(
+def remove_dropout_nodes_inplace(
+    nncf_graph: NNCFGraph,
+    dropout_metatypes: List[OperatorMetatype],
+) -> None:
+    """
+    Removes the Dropout nodes from the provided NNCFGraph instance and
+        connects droput previous node with dropout next nodes inplace
+        for each dropout node.
+
+    :param nncf_graph: NNCFGraph instance for the transformation.
+    :param dropout_metatypes: List of backend-specific Dropout metatypes.
+    """
+    if not dropout_metatypes:
+        return
+
+    nodes_to_drop = []
+    for node in nncf_graph.get_all_nodes():
```

Comment:
Why not to use `nncf_graph.get_nodes_by_metatypes`?

Reply:
Done